.github/workflows/pep8.yaml

```diff
@@ -0,0 +1,40 @@
+name: PEP8 linter
```

Comment:
Can we please use the same [version as in the core](https://github.com/viur-framework/viur-core/blob/develop/.github/workflows/pep8.yaml), with the GH annotations?